components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/servlet/SAMLSSOProviderServlet.java

```diff
@@ -1191,6 +1193,40 @@ private void handleAuthenticationReponseFromFramework(HttpServletRequest req, Ht
         }
     }
 
+    /**
+     * Store SamlSSOTokenId In session context.
+     *
+     * @param samlssoTokenId       samlssoTokenId.
+     * @param authenticationResult authenticationResult.
```

Comment:
Start the param description with an uppercase letter and better to add it like "SAML SSO token IS" and "Authentication result". WDYT?

Reply:
changed to Authentication Result. But samlssoTokenId is a cookie name.